packages/perseus/src/util/parse-perseus-json/general-purpose-parsers/string-to-number.ts

```diff
@@ -0,0 +1,17 @@
+import type {PartialParser} from "../parser-types";
+
+export const stringToNumber: PartialParser<string | number, number> = (
```

Comment:
Sadly, there is more than one place where Perseus data contains both actual numbers and stringified numbers. 

No action needed.